osu.Game/Graphics/UserInterface/ScreenTitle.cs

```diff
@@ -72,27 +79,29 @@ protected ScreenTitle()
                         {
                             t.Anchor = Anchor.Centre;
                             t.Origin = Anchor.Centre;
+                            t.Margin = new MarginPadding { Horizontal = 5 }; // compensates for osu-web sprites having around 5px of whitespace on each side
                         }),
-                        titleText = new OsuSpriteText
+                        TitleText = new OsuSpriteText
                         {
                             Anchor = Anchor.Centre,
                             Origin = Anchor.Centre,
                             Font = OsuFont.GetFont(size: 20, weight: FontWeight.Bold),
-                            Margin = new MarginPadding { Bottom = text_offset }
+                            Margin = new MarginPadding { Vertical = text_padding }
                         },
-                        new Circle
+                        Separator = new Circle
                         {
                             Anchor = Anchor.Centre,
                             Origin = Anchor.Centre,
-                            Size = new Vector2(4),
+                            Size = new Vector2(5),
                             Colour = Color4.Gray,
+                            Margin = new MarginPadding { Top = 3 } // compensation for osu-web using a font here making the circle appear a bit lower
                         },
```

Comment:
I checked and it should be possible to fully mirror web implementation here:

```csharp
                        Separator = new OsuSpriteText
                        {
                            Anchor = Anchor.Centre,
                            Origin = Anchor.Centre,
                            Font = OsuFont.GetFont(size: 20),
                            Colour = Color4.Gray,
                            Margin = new MarginPadding { Vertical = text_padding },
                            Text = "\u2022"
                        },
```

Reply:
For some reason I thought we can't use unicode chars like this here (don't know why, haven't even tried tbh), thanks